### ISSUE TITLE
NOT MANTAINED: This project seems to be unmaintained

### ISSUE THREAD

**Issue by godjangollc** [MaistrenkoAnton/django-material-admin#181]
Since there has been no new commit since two years ago, I will assume this project is unmaintained, if this is not the case please let me know, there are a few minor issues that need to be used, I opened a PR for solving it(#180 ) before noticing that there was another PR solving the same issue, and the other when fields are readonly the text overlap the verbose name set in django admin... If this project is not maintained anymore I will eventually be fixing those issues and maintaining this project in my own fork at https://github.com/godjangollc/godjango-material-admin. The pip repo will be called godjango-material-admin... If there is any issue with that please answer this issue. (I saw conversations are not allowed that's why I used a issue instead) Best regards and thanks for this awesome project.

**Comment by luigibertaco** [comment]
Hey, did you end up creating the new repo? is it public? I am looking forward to replacing this dependency with a maintained alternative in a personal project.

Willing to contribute as well.

**Comment by godjangollc** [comment]
Yep sorry for the late response, I got a notification about this repo but I didnt knew was someone replying to my post. The new repo it is at https://github.com/godjangollc/godjango-material-admin

**Comment by godjangollc** [comment]
Open any PR there and any issue I'll try to fix them asap. I am thinking on implementing the controls of https://mui.com/ in the version hosted there.

**Comment by eddyojb88** [comment]
Hi @godjangollc is your repo pip installable?

**Comment by godjangollc** [comment]
Not yet...

**Comment by eddyojb88** [comment]
I am using django-unfold, which looks very neat and professional - recommend checking it out.

**Comment by devdinho** [comment]
I forked it and it is now available on pypi. Feel free to participate in the project and make pull requests.
This is the repository link:

https://github.com/devdinho/materialdash

I still need to improve pypi readme.

I also changed the name of the app to put it among the installed Django apps to "materialdash".

**Comment by Eternal254** [comment]
@freitasanderson1 brooo, I've been having problems with material admin since days ago, and with yours it worked again, thanks <3.

**Comment by devdinho** [comment]
You're welcome, my friend @Eternal254. If you have any problems or bugs related to materialdash, feel free to report the error and even contribute to the fix.
